website/content/en/docs/olm-integration/cli-overview.md

```diff
@@ -1,34 +1,51 @@
 ---
 title: Operator Bundle Tooling in Operator SDK
 linkTitle: CLI Overview
```

Comment:
```suggestion
linkTitle: Operator Bundle Tooling
```

Reply:
CLI overview is not clarifying what it is. Note that we have the CLI docs as well which are gen automatically.

Reply:
All OLM-related subcommands are discussed here, so I've updated the `title` to be more clear. The `linkTitle` is relative to the "OLM Integration" section so the context is clear IMO.

Reply:
The linked title is what appears in the `menu`, so, imo change the title do not solve it.